index.html

```diff
@@ -148,6 +148,35 @@ <h2>Skills</h2>
         </ul>
     </li>
   </ul>
+  <footer>
+     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+     <form action="https://formspree.io/f/mayvbgnj" method="POST" class="contact-form">
+        <input type="text" placeholder="Full Name" class="text" maxlength="30" required>
+        <input type="email" class="text" placeholder="Email address" pattern="/^[a-zA-Z0-9.!#$%&amp;'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
+        <textarea class="areatext" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ ] Kindly note that the `Enter text here` text should be a placeholder. example :point_down: 

```html
        <textarea class="areatext" maxlength="500" placeholder="Enter text here" required></textarea>
```